### ISSUE TITLE
Poor perfromance on Adreno 306 and 308

### ISSUE THREAD

**Issue by roninsi** [pixijs/pixi.js#6180]
Hello, while checking base performance on multiple devices for our future project I found two devices where my simple test would render basically unplayable. Reducing test to 

 const app = new PIXI.Application({
               autoResize: true, backgroundColor: 0x1099bb,
 });
 document.body.appendChild(app.view);

and later updating on resize  app.renderer.resize(777,360);

still doesn't reach more than 40 FPS. Using Pixi-legacy and canvas, the performance is much better. 

Profiling doesn't show much, most of the time is spent on System (self) with no further details.

![image](https://user-images.githubusercontent.com/21121476/66991205-e5702180-f0c7-11e9-90a2-91eba72fe953.png)

Devices were Huawei MediaPAd T3 10 and Samsung Galaxy Tab A(SM-T550), both runing android 7.  Browsers tested were Chrome and Firefox.

Any sugestion what I am doing wrong or what setting could improve performance?

**Comment by eXponenta** [comment]
Andreno 308, 306 is a very slow.

It is not problem of PIXI, it is problem of devices.
System call is very heavyweight.

Usung screen image preview on debuger decrease performnace too. Disable it and capture performance log, then include it as attachment, we we try resolve your problem.

**Comment by roninsi** [comment]
Hi, thank you for looking at this and I apologise for late reply.

I did as you suggested and performance in logs has improved. To bring it closer to the performance edge I used full scale render this time.

I am attaching three files, all from Huawei MediaPad T3 10, rendered at 1555x720.

[Profiles.zip](https://github.com/pixijs/pixi.js/files/3750845/Profiles.zip)

First one is "empty" demo without any graphics added, just background colour as in initial post.

Second one is "slot" from examples. I had to remove blurring effect from symbols, since those instantly brought down performance to 15FPS.

Last one is "custom" test made on top of slot example, with more symbols, background image instead of colour and proper masking instead of black bars. Addition of masking doesn't seem to do any performance hit. Background had to be composited together, cached and masked to avoid offscreen rendering due to oversized background. All textures were packed in single sprite.

While this does run at 50FPS (40 without bitmapcaching), I am afraid that this would still drop below 30 with more items and animations added. Since those three samples are rendered at full resolution, I still have an option to render in half scale if needed. Are there any other suggestions what can be done to bring performance up?
